include/pnet_api.h

```diff
@@ -1725,6 +1725,16 @@ PNET_EXPORT int pnet_diag_remove(
  *     0x2000              | Show CFG information.
  *     0x4000              | Show scheduler information.
  *     0x8000              | Show I&M data.
+ *
```

Comment:
Was this intended? I can't tell from the commit message.

Reply:
Updated commit message